browser/resources/settings/brave_overrides/page_visibility.ts

```diff
@@ -57,7 +57,7 @@ function getPageVisibility () {
     // future-proof chromium actually defining something,
     ...chromiumPageVisibility,
     // overrides
-    a11y: false,
+    a11y: true,
```

Comment:
do we need to override this property anymore?

Reply:
nice catch!